frontend/public/locales/zh/public.json

```diff
@@ -1856,13 +1856,13 @@
   "ClusterRole": "集群角色",
   "ClusterRoles": "集群角色",
   "SelfSubjectAccessReview": "SelfSubjectAccessReview",
-  "SelfSubjectAccessReviews": "SelfSubjectAccessReviews",
+  "SelfSubjectAccessReviews": "selfsubjectaccessreviews",
   "SelfSubjectReview": "SelfSubjectReview",
   "SelfSubjectReviews": "SelfSubjectReviews",
   "ResourceAccessReview": "ResourceAccessReview",
   "ResourceAccessReviews": "ResourceAccessReviews",
   "LocalResourceAccessReview": "LocalResourceAccessReview",
-  "LocalResourceAccessReviews": "LocalResourceAccessReviews",
+  "LocalResourceAccessReviews": "localresourceaccessreviews",
```

Comment:
Should this be CapitalCase?

Reply:
> Should this be CapitalCase?

Hmm, not sure why the translators made this change. But I will open a bug for the investigation.